features/access_management.md

```diff
@@ -0,0 +1,24 @@
+# Access management
+TokenD is based on a permissioned blockchain that limit the parties (it can be wether another person or external service) who can transact on the blockchain and that set who can serve the network by writing new blocks into the chain.
+
+Therefore Tokend has pack of functionalities that help us to distribute the rights. In order to use them, you need to be familiar with 2 key entities: **accounts** and **signers**
+
+### Accounts
+
+Accounts are the central data structure in TokenD. First of all, an account is a human-readable name that is stored on the blockchain. An account is required to transfer or otherwise push a transaction to the blockchain. Accounts are identified by a public key and saved in the ledger. Another data in the ledger, such as assets, offers, or KYC data are owned by a particular account.
+
+TokenD differs the type of account to master's and user's [accounts](/tech/key_entities/accounts#account-type). The owner of master account is allowed to perform a lot of operations that will have impact on the whole system.
+
+Detailed description of system's security is available [here](/tech/security).
+
+### Signers
+
+The account owner can assign signers to his employees or some services. Signer is the main concept used for authorization. Therefore, the signer will be able to perform transaction on read private data of this account from any of the services signature of one of the signers of that account must be provided. For more info, see [here](/tech/key_entities/signer).
+

```

Comment:
remove from....provided

Reply:
Each account (especially master) can be managed by different parties. Here the *signer* definition comes. *Signer* is an entity that is able to perform transactions on read private data of this account depending on it's (signer's) policies